src/styles/blocks/main/features.scss

```
@@ -0,0 +1,103 @@
+.features {
+  $maxWidthCard: 370px;
+  padding: 0 $sidePaddingMobile;
+  margin-bottom: 95px;
```

```suggestion
  margin-bottom: 95px;
```
We cannot use outer spacings for BEM-blocks